Domain/Aggreagtes/UserAggregate/User.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Domain.Aggreagtes.ApplicantAggregate;
+using Domain.Aggreagtes.StudentAggregate;
+using Domain.Common.Contracts;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Aggreagtes.UserAggregate
+{
+    public class User : AuditableEntity, IAggregateRoot
+    {
+        public string UserName { get; private set; }
+        public string EmailAddress { get; private set; } = default!;
+        public string PasswordHash { get; private set; }
+        public string PasswordSalt { get; private set; }
+        public virtual Applicant? Applicant { get;  set; }
+        public virtual Student? Student { get; set; }
```

Comment:
why have student and applicaant in the user table. Does that mean there will be staff here too?
There is already a natural keep that links a record to the student or applicant that owns it. That is the email address. There is no need for any object referencing between them.